### PR TITLE
fix: additional whitespace in user selector

### PR DESCRIPTION
**Pull request by edalex-yinzi**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [x] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change
Fix additional whitespace in user selector by adding two classes to the root div for wizards in AdvSearchPanel.

Solution found in [here](https://stackoverflow.com/questions/51445959/unexpected-empty-space-using-flexbox).

https://user-images.githubusercontent.com/92769668/144966006-e633327d-04bf-4014-88e9-c61d14f91900.mp4

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ export const AdvancedSearchPanel = ({
             currentValues,
             onChangeHandler
           ).map((e) => (
-            <Grid key={e.props.id} item>
+            <Grid key={e.props.id} item container direction="column">
```
Comment (edalex-ian) [inline react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx:133]: Hmmmmmmm.... Not sure on this one @edalex-yinzi , you normally have either `item` or `container`... Not both. :thinking:

Comment (PenghaiZhang) [reply react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx]: I think a nested container is OK. Here is an [example](https://github.com/mui-org/material-ui/pull/12313/files)

File: react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx [hunk 2]
```diff
@@ -130,7 +130,7 @@ export const AdvancedSearchPanel = ({
             currentValues,
             onChangeHandler
           ).map((e) => (
-            <Grid key={e.props.id} item>
+            <Grid key={e.props.id} container direction="column">
```
Comment (PenghaiZhang) [inline react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx:133]: It should really be an `item` at least.

Comment (edalex-yinzi) [reply react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx:133]: And we do need its margin style anyway, I will add it back.

Comment (PenghaiZhang) [reply react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx:133]: If you make this `item` a nested `container`, whatever inside it should be wrapped by an `item`. 
I am not quite what styles you need, but if they are quite simple, perhaps an inline style is sufficient ??

**Review by edalex-ian — APPROVED** [review]
Makes sense to say by default each WizardControl is provided with 100% width, so, sure. :wink: :+1:

**Review by PenghaiZhang — APPROVED** [review]

**Pull request merged**